### ISSUE TITLE
Bug in beta with mailru

### ISSUE THREAD

**Issue by sp31415t1** [rclone/rclone#5451]
#### The associated forum post URL from `https://forum.rclone.org`
https://forum.rclone.org/t/bug-in-beta-with-mailru/25218

#### What is the problem you are having with rclone?
I tried to use the beta version which can be unstable.
I don't know if there is any work in progress at the moment which may be the cause.
I also tested the same command with yandex, mega and webdav without any issue.
I'm submitting, because it may be an unknown regression.
in 1.54.1 , 1.55.0.and 1.55.1, it was functional.
with 1.56 beta and a normal (not on-the-fly) backend, rclone about is functional

#### What is your rclone version (output from `rclone version`)
rclone v1.56.0-beta.5580.aab29353d
- os/version: debian 10.10 (64 bit)
- os/kernel: 4.19.0-17-amd64 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.16.5
- go/linking: static
- go/tags: none

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
linux debian stable

#### Which cloud storage system are you using? (e.g. Google Drive)
mailru

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
```
rclone -vv --config /notfound about :mailru: --mailru-user hidden@mail.ru --mailru-pass hidden --mailru-speedup-enable=false
```

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
```
2021/07/08 16:27:16 DEBUG : rclone: Version "v1.56.0-beta.5580.aab29353d" starting with parameters ["rclone" "-vv" "--config" "/notfound" "about" ":mailru:" "--mailru-user" "hidden@mail.ru" "--mailru-pass" "hidden" "--mailru-speedup-enable=false"]
2021/07/08 16:27:16 DEBUG : Creating backend with remote ":mailru:"
2021/07/08 16:27:16 DEBUG : :mailru: detected overridden config - adding "{mSzhq}" suffix to name
2021/07/08 16:27:16 DEBUG : Config is memory-only - using defaults
2021/07/08 16:27:16 INFO  : []: Valid token not found, authorizing.
2021/07/08 16:27:16 NOTICE: Can't save config "token" = "{\"access_token\":\"hidden\",\"token_type\":\"Bearer\",\"refresh_token\":\"hidden\",\"expiry\":\"2021-07-08T17:27:16.743118113+02:00\"}" for on the fly backend ":mailru"
2021/07/08 16:27:16 DEBUG : :mailru{mSzhq}: Saved new token in config file
2021/07/08 16:27:16 Failed to create file system for ":mailru:": empty token found - please run "rclone config reconnect :mailru{mSzhq}:"
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
This does indeed look like a bug!

Can you check this out please?

[v1.56.0-beta.5586.2885971fa.fix-5451-mailru-config](https://beta.rclone.org/branch/fix-5451-mailru-config/v1.56.0-beta.5586.2885971fa.fix-5451-mailru-config/) on branch [fix-5451-mailru-config](https://github.com/rclone/rclone/tree/fix-5451-mailru-config) (uploaded in 15-30 mins)

If it works for you I'll squeeze it into 1.56

**Comment by sp31415t1** [comment]
It looks good to me.
Thank you

**Comment by ncw** [comment]
Thanks for testing.

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.56
